### PR TITLE
feat: make pipeline support instanceCap field

### PR DESCRIPTION
**Pull request by zoumo**
feat: make pipeline support instanceCap field
fix: containerCap and instanceCap judgement bug

### CODE REVIEW

**Comment by iocanel** [comment]
@zoumo: Thanks for the PR, I add a minor change request, so that its complete! Thanks again!

File: src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java [hunk 1]
```diff
@@ -655,13 +655,13 @@ private boolean addProvisionedSlave(PodTemplate template, Label label) throws Ex
         PodList namedList = client.pods().withLabels(labelsMap).list();
         List<Pod> namedListItems = namedList.getItems();
 
-        if (slaveListItems != null && containerCap < slaveListItems.size()) {
```
Comment (zoumo) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java]: Thanks! I think this judgement is not correct. if container capacity == slaveListIterms.size(), cloud should not be able to provision slaves any more

Comment (zoumo) [inline src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java]: Do you have any plan to fix this?

**Comment by zoumo** [comment]
@iocanel Hi, I have updated config.jelly. request review.
Could you tell me what this file is doing? I can not find the view in jenkins defined in this config

**Comment by iocanel** [comment]
@zoumo: Thanks! This is used by the script generator.

**Pull request merged**